content/lib/Pdf/Content/Parser.hs

```diff
@@ -43,8 +44,15 @@ parseContent = do
   skipSpace
   (Parser.endOfInput >> return Nothing) <|>
     fmap Just (fmap Obj parseObject <|>
+              parseInlineImage <|>
                fmap (Op . toOp) (Parser.takeWhile1 isRegularChar))
 
+parseInlineImage :: Parser Expr
+parseInlineImage = do
+  Parser.string "ID"
+  Parser.manyTill Parser.anyChar (Parser.string "EI")
```

Comment:
This is very suspicious. What if image data contains EI?

Reply:
might be interesting: https://github.com/mozilla/pdf.js/pull/16461

Reply:
I agree, it worried me too.